index.html

```diff
@@ -4,18 +4,58 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <!-- Add a link to your custom CSS here -->
-
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@200;500&family=Roboto:ital,wght@1,100;1,300&display=swap" rel="stylesheet">
     <title>Responsive grid project</title>
 </head>
+<body class="home">
+<header class="flex j-b a-i">
+    <div class="flex a-i">
 
-<body>
-
-    <!-- Add your markup here -->
+        <img class="logo" src="https://images.unsplash.com/photo-1588421357574-87938a86fa28?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1470&q=80" alt="">
 
-
-    <!-- Please leave this in to credit the creators of this design -->
-    <p>This website design was created by Wix.com, and is used here for strictly educational purposes.</p>
+        <h2>
+          <span class="name">Khesiwe Dube</span>  
+        <span class="text-sm">/ &nbsp;Software Developer Trainee</span></h2>
+    </div>
+    <nav>
+        <ul>
+            <li> <a class="nav-bar" href="">About Me</a></li>
+            <li> <a  href="./resume.html">Resume</a></li>
+            <li> <a  href="">Projects</a></li>
+            <li> <a  href="">Contact</a></li>
+        </ul>
+    </nav>
+</header>
+    <main>
+        <div class="main-box">
+    <section class="text-box"> 
+        <img class="aside-img" src="https://scontent.fman4-1.fna.fbcdn.net/v/t1.6435-9/75241133_466332603985792_980425490466603008_n.jpg?_nc_cat=105&ccb=1-7&_nc_sid=174925&_nc_ohc=JC3TjgCm0X0AX-YCywI&_nc_ht=scontent.fman4-1.fna&oh=00_AT8NgcunjLOZ-qHMUKeB_XIFJetyRICHwBwjnxiUHaMzXQ&oe=6322AD7C" alt="image">
```

Comment:
This image isn't available but it looked nice when it was there. More descriptive alt text would be good, something like Headshot of Khesiwe Dube.